consensus/quorum/quorum.go

```diff
@@ -8,26 +8,24 @@ import (
 	bls_cosi "github.com/harmony-one/harmony/crypto/bls"
 	"github.com/harmony-one/harmony/numeric"
 	"github.com/harmony-one/harmony/shard"
-	"github.com/harmony-one/harmony/staking/slash"
-	// "github.com/harmony-one/harmony/staking/effective"
 )
 
 // Phase is a phase that needs quorum to proceed
 type Phase byte
 
 const (
-	// Prepare ..
-	Prepare Phase = iota
+	// Announce ..
+	Announce Phase = iota
```

Comment:
why change it?

Reply:
Yea, prepare and commit are the two phases in FBFT, announce is not a phase, it's part of prepare phase..

Reply:
will change name back